Sources/nest/Commands/RunCommand.swift

```diff
@@ -0,0 +1,174 @@
+        
+        guard !arguments.isEmpty else {
+            logger.error("`owner/repository` is not specified.", metadata: .color(.red))
+            return
+        }
+        guard arguments[0].contains("/") else {
+            logger.error("Invalid format: \(arguments), expected owner/repository", metadata: .color(.red))
+            return
+        }
+        
+        let reference = arguments[0]
+        let subcommands: [String] = if arguments.count >= 2 {
+            Array(arguments[1...])
+        } else {
+            []
+        }
+        guard let target = nestfileController.fetchTarget(reference: reference, nestfile: nestfile),
+              let expectedVersion = target.version
+        else {
+            // While we could execute with the latest version, the bootstrap subcommand serves that purpose.
+            // Therefore, we return an error when no version is specified.
+            logger.error("Failed to find expected version in nestfile", metadata: .color(.red))
+            return
+        }
+        
+        guard let installTarget = InstallTarget(argument: reference),
+              case let .git(gitURL) = installTarget,
+              let gitVersion = GitVersion(argument: expectedVersion)
+        else {
+            return
+        }
+
+        guard let binaryRelativePath = try await resolveBinaryRelativePath(
+            didAttemptInstallation: false,
+            noInstall: noInstall,
+            reference: reference,
+            version: expectedVersion,
+            target: target,
+            gitURL: gitURL,
+            gitVersion: gitVersion,
+            nestInfoController: nestInfoController,
+            executableBinaryPreparer: executableBinaryPreparer,
+            artifactBundleManager: artifactBundleManager,
+            logger: logger
+        ) else {
+            logger.error(
+                "Failed to find binary path, likely because it's not installed. Please try without the --no-install option or run the bootstrap command.",
+                metadata: .color(.red)
+            )
+            return
+        }
+
+        _ = try await NestProcessExecutor(logger: logger, logLevel: .info)
+            .execute(
+                command: "\(nestDirectory.rootDirectory.relativePath)\(binaryRelativePath)",
+                subcommands
+            )
+    }
+
+    private func resolveBinaryRelativePath(
+        didAttemptInstallation: Bool,
+        noInstall: Bool,
+        reference: String,
+        version: String,
+        target: Nestfile.Target,
+        gitURL: GitURL,
+        gitVersion: GitVersion,
+        nestInfoController: NestInfoController,
+        executableBinaryPreparer: ExecutableBinaryPreparer,
+        artifactBundleManager: ArtifactBundleManager,
+        logger: Logger
+    ) async throws -> String? {
+        guard let binaryRelativePath = nestInfoController.fetchCommand(reference: reference, version: version)?.binaryPath
```

Comment:
Use if statement instead of guard statement.

Reply:
Thanks!
I fixed it together here. (https://github.com/mtj0928/nest/pull/36/commits/ff049a9bba8fa8e08633cef1c49581f1e1865698)